primo/opt_model/tests/test_result_parser.py

```diff
@@ -643,7 +643,7 @@ def test_get_efficiency_metrics(get_efficiency_calculator):
     assert all(
         [
             list(efficiency_metric_output.iloc[0, :].values)[i]
-            == pytest.approx([1, 10.0, 20.0, 0.0, 20][i])
+            == pytest.approx([2, 10.0, 20.0, 0.0, 20][i])
             for i in range(5)
         ]
```

Comment:
So unfortunate, I had these changes already made before we went to the 1,2,3,4,5... convention... hopefully it was easy enough to follow.